tsl/src/bgw_policy/continuous_aggregate_api.c

```diff
@@ -7,12 +7,27 @@
 #include <postgres.h>
 
 #include <jsonb_utils.h>
-
+#include <miscadmin.h>
+#include <utils/builtins.h>
 #include "bgw_policy/continuous_aggregate_api.h"
 #include "bgw_policy/job.h"
+#include "bgw/job.h"
+#include "continuous_agg.h"
+#include "dimension.h"
+#include "hypertable_cache.h"
+#include "policy_utils.h"
 
-#define POLICY_REORDER_PROC_NAME "policy_reorder"
+#define POLICY_REFRESH_CAGG_PROC_NAME "policy_refresh"
```

Comment:
I couldn't find this function in the PR. Does it already exist? Shouldn't the policy API check that a function exists and fail if it doesn't? I guess this should be common for all policies?

Reply:
corrected the name. Would have been caught when we actually try to execute. That is not yet integrated here.